plugin/storage/grpc/shared/grpc_handler.go

```diff
@@ -244,45 +241,16 @@ func (*GRPCHandler) sendSpans(spans []*model.Span, sendFn func(*storage_v1.Spans
 
 func (s *GRPCHandler) Capabilities(context.Context, *storage_v1.CapabilitiesRequest) (*storage_v1.CapabilitiesResponse, error) {
 	return &storage_v1.CapabilitiesResponse{
-		ArchiveSpanReader:   s.impl.ArchiveSpanReader() != nil,
-		ArchiveSpanWriter:   s.impl.ArchiveSpanWriter() != nil,
+		ArchiveSpanReader:   s.impl.SpanReader() != nil,
+		ArchiveSpanWriter:   s.impl.SpanWriter() != nil,
```

Comment:
@yurishkuro should we remove these from the capabilities response altogether?

Reply:
yes

Reply:
we can mark them deprecated in the .proto definition and return `false` here